### PR TITLE
Await task worker rpc init

### PR DESCRIPTION
**Pull request by disambiguator**
Without this, we cannot be sure that the task worker has finished initializing before returning from the main
runtime initialize function

### CODE REVIEW

File: zaplib/web/wasm_runtime.ts [hunk 1]
```diff
@@ -834,24 +834,24 @@ export const initialize: Initialize = (initParams) => {
...
+            wasmExports = instance.exports as WasmExports;
+            initThreadLocalStorageMainWorker(wasmExports);
+            const tlsAndStackData =
+              makeThreadLocalStorageAndStackDataOnExistingThread(wasmExports);
+            wasmAppPtr = wasmExports.createWasmApp();
+            // The calls above are safe when wasm isn't online yet, but after that let's
+            // wrap for safety.
+            wasmExports = wrapWasmExports(wasmExports);
+
+            await rpc.send(
```
Comment (stevekrouse) [inline zaplib/web/wasm_runtime.ts:854]: This is one new await

File: zaplib/web/wasm_runtime.ts [hunk 2]
```diff
@@ -865,16 +865,19 @@ export const initialize: Initialize = (initParams) => {
...
-            });
-        }, reject);
+            );
+            canvasData.onScreenResize();
+            if (initParams.defaultStyles) {
+              removeLoadingIndicator();
+            }
+            initialized = true;
+
+            await taskWorkerRpcInit;
```
Comment (stevekrouse) [inline zaplib/web/wasm_runtime.ts:875]: This is a second new await

**Comment by stevekrouse** [comment]
I probably don't have enough context, but looks alright to me 🙃

File: zaplib/web/wasm_runtime.ts (hunk 1 above)
Comment (disambiguator) [reply zaplib/web/wasm_runtime.ts:854]: This first new await is just a conversion from a promise to await syntax. pure refactor

**Pull request closed without merging**